pyproject.toml

```diff
@@ -89,6 +89,7 @@ extend-exclude = [
 "examples/*" = ["T20"]
 "docs/broken_links.py" = ["T20"]
 "mlflow/*" = ["PT018"]
+"tests/pyfunc/docker/test_docker_flavors.py" = ["F401"] # For pytest fixture imports
```

Comment:
```suggestion
```

We can add `noqa: F401` on lines violating this rule.

Reply:
yea I did that at first but was a bit tedious to repeat it for many lines....